crm_phonecall_category/views/crm_phonecall_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record id="crm_case_phone_tree_view" model="ir.ui.view">
+    <field name="name">Show category</field>
+    <field name="model">crm.phonecall</field>
+    <field name="inherit_id" ref="crm.crm_case_phone_tree_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='categ_id']" position="attributes">
+            <attribute name="invisible">False</attribute>
+            <attribute name="widget">selection</attribute>
+            <attribute name="domain">[
+                ('object_id.model', '=', 'crm.phonecall'),
+            ]</attribute>
+        </xpath>
+    </field>
+</record>
+
+<record id="crm_case_inbound_phone_tree_view" model="ir.ui.view">
+    <field name="name">Show category</field>
+    <field name="model">crm.phonecall</field>
+    <field name="inherit_id" ref="crm.crm_case_inbound_phone_tree_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='categ_id']" position="attributes">
+            <attribute name="invisible">False</attribute>
```

Comment:
If this is the inbound tree, isn't the category already set and you shouldn't change it?